cpp/models/abm/person.h

```diff
@@ -422,7 +422,8 @@ class Person
 
 private:
     LocationId m_location; ///< Current Location of the Person.
-    std::vector<uint32_t> m_assigned_locations; /**! Vector with the indices of the assigned Locations so that the 
+    LocationType m_location_type; ///< Type of the current Location.
```

Comment:
This is not a change - it only makes explicit that the persons holds its locations type.

Reply:
Currently it is necessary. Otherwise, the functions using it (mainly migration rules) require the type or World&. But I claim this is out of scope and should be done in another issue (if you want that).